evm-template/runtime/src/apis.rs

```diff
@@ -1,664 +0,0 @@
-use frame_support::{
```

Comment:
it has to reside with construct_runtime, otherwise polkadot js cannot see the metadata properly. I want to move it back to a separate file again but this fix is urgent

Reply:
also the explanation was on the PR body: 

> impl_runtime_api macro has to be in the same file with construct runtime for now (we can find a way and separate those again once we figure out how, but right now let's focus on the fix)

I think you missed it